examples/guides/services/hospital-service/daos/healthcare_dao.bal

```diff
@@ -0,0 +1,55 @@
+type DoctorNotFoundError error<string>;
```

Comment:
Let's add license header for all the files.

Reply:
Added